lib/scraper.js

```diff
@@ -127,9 +187,24 @@ Scraper.prototype.prepare = function prepare () {
 
 Scraper.prototype.load = function load () {
 	var self = this;
-	return Promise.map(self.originalResources, function loadResource (resource) {
-		return self.loadResource(resource);
+	_.forEach(self.originalResources, function loadResource (resource) {
+		self.loadResource(resource);
 	});
+	return self.waitForLoad();
+};
+
+Scraper.prototype.waitForLoad = function waitForLoad (previousCount) {
```

Comment:
As for me, it looks messy. 
I don't like that `count` (number) is compared with `previousCount` (undefined at the first step)

Can we somehow simplify it?